servers/quarkus-server/src/main/java/org/projectnessie/server/config/VersionStoreConfig.java

```diff
@@ -25,10 +25,14 @@
 public interface VersionStoreConfig {
 
   @RegisterForReflection
-  public enum VersionStoreType {
+  enum VersionStoreType {
+    @Deprecated
+    IN_MEMORY,
+    @Deprecated
+    JGIT,
     DYNAMO,
     INMEMORY,
```

Comment:
having IN_MEMORY and INMEMORY is quite confusing. Maybe rename INMEMORY to INMEMORY_V2 or something until the other one is gone?

Reply:
Similar to the above - `IN_MEMORY` is going away.